.github/workflows/acpi-validation.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will run mock-acpi compose on GH runner and run validator tool
+
+name: Validate ACPI
+
+on: # yamllint disable-line rule:truthy
+  workflow_call:
+
+jobs:
+  validate-acpi:
+    name: Validate ACPI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+
+      - name: install hatch
+        run: |
+          python -m pip install --upgrade pip
+          pip install hatch
+
+      - name: Run mock-acpi compose
+        working-directory: manifests/compose/mock-acpi
+        run: |
+          docker compose up -d
+
+      - name: Wait for services to be ready
+        run: |
+          echo "Waiting for services to start..."
+          sleep 60
```

Comment:
from robust part of view, can we make a timeout? as a loop with condition to wait for compose up and timeout break as failure?

Reply:
Ideally, it should be warm-up time for Kepler to start producing metrics. I have updated this step